cartographer/mapping_2d/local_trajectory_builder.cc

```diff
@@ -121,26 +122,29 @@ LocalTrajectoryBuilder::AddRangeData(const common::Time time,
 
   if (num_accumulated_ >= options_.scans_per_accumulation()) {
     num_accumulated_ = 0;
+    const transform::Rigid3d gravity_alignment = transform::Rigid3d::Rotation(
+        extrapolator_->EstimateGravityOrientation(time));
     return AddAccumulatedRangeData(
-        time, sensor::TransformRangeData(accumulated_range_data_,
-                                         tracking_delta.inverse()));
+        time,
+        // Transforms 'accumulated_range_data_' into a frame where gravity
```

Comment:
Remove the comment? The function name now already better reflects this and the comment misses the part that the call also filters the range data.